h/static/scripts/annotator/guest.coffee

```diff
@@ -1,15 +1,7 @@
-$ = require('jquery')
 Annotator = require('annotator')
-Channel = require('jschannel')
+$ = Annotator.$
 
-require('diff-match-patch')
```

Comment:
These dependencies belong to various Annotator plugins. I have browserified those libraries once in the past...  let me look those up.